test/orange/univ/SimpleUnitTracker.test.cc

```diff
@@ -967,6 +973,22 @@ TEST_F(FiveVolumesTest, TEST_IF_CELERITAS_DOUBLE(heuristic_init))
     }
 }
 
+TEST_F(ConeTest, safety)
+{
```

Comment:
Oh crap, I forgot that the Geant input gets scaled into the native unit system (unlike the .org.json files which are precalculated). This test is with the CLHEP unit system which scales everything by a factor of 10. I think it would be better to build a geometry  with the InputBuilder test...  sorry 😞